java/arcs/core/storage/StorageProxy.kt

```diff
@@ -71,7 +71,7 @@ class StorageProxy<Data : CrdtData, Op : CrdtOperationAtTime, T> private constru
     val dispatcher: CoroutineDispatcher
         get() = scheduler.asCoroutineDispatcher()
 
-    private val log = TaggedLog { "StorageProxy" }
+    private val log = TaggedLog { "StorageProxy" }.withSuffix { "(key=$storageKey)" }
```

Comment:
This we unfortunately can’t merge as storage key is not safe for non debug logging. Unless we enhance the suffix thing to allow specifying log level?

Reply:
You could make a special .debugSuffix() that only appears it to log.debug() messages perhaps.